### ISSUE TITLE
Fails to scale when there are more than 50 nodes in the ASG

### ISSUE THREAD

**Issue by mikkeloscar** [hjacobs/kube-aws-autoscaler#40]
We scaled a cluster to 60 nodes and the autoscaler failed to scale down:

```
2017-11-02 22:34:17,332 ERROR: Failed to autoscale
Traceback (most recent call last):
  File "/kube_aws_autoscaler/main.py", line 387, in main
    include_master_nodes=args.include_master_nodes, dry_run=args.dry_run)
  File "/kube_aws_autoscaler/main.py", line 338, in autoscale
    nodes_by_asg_zone = get_nodes_by_asg_zone(autoscaling, all_nodes)
  File "/kube_aws_autoscaler/main.py", line 124, in get_nodes_by_asg_zone
    response = autoscaling.describe_auto_scaling_instances(InstanceIds=list(instances.keys()))
  File "/usr/lib/python3.5/site-packages/botocore/client.py", line 253, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/lib/python3.5/site-packages/botocore/client.py", line 557, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (ValidationError) when calling the DescribeAutoScalingInstances operation: The number of instance ids that may be passed in is limited to 50
```

**Comment by hjacobs** [comment]
@mikkeloscar can you check what version of the autoscaler is running in your cluster? This problem should have been fixed a long time ago, see #25

**Comment by hjacobs** [comment]
Oh, damn, this was not really fixed as I see now. I will do a PR.
